torch/_torch_docs.py

```diff
@@ -3117,7 +3117,7 @@ def merge_dicts(*dicts):
 .. warning::
     This function's name is a misnomer. It actually rounds the
     quotient towards zero instead of taking its floor. This behavior
-    will be deprecated in a future PyTorch release.
+    is deprecated and will change in a future PyTorch release.
```

Comment:
Let's start this paragraph with the most important bit: 

"torch.floor_divide is deprecated and will be removed in a future PyTorch release. Its name is a misnomer because it actually rounds the quotient towards zero instead of taking its floor. To keep the current behavior use torch.div() with the 'trunc' rounding mode. To actually perform floor division use torch.div() with the 'floor' rounding mode."